javascript/classes.json

```diff
@@ -507,10 +507,12 @@
               "version_added": "53"
             },
             "safari": {
-              "version_added": false
+              "version_added": false,
```

Comment:
BCD accepts Safari 14 as a value and it's tracked as a beta version: https://github.com/mdn/browser-compat-data/blob/master/browsers/safari.json#L170. You can remove the note and set `"version_added": "14"`